omero/developers/Matlab.rst

```diff
@@ -51,7 +51,7 @@ As described under |OmeroClients|, there are several ways to configure your
 connection to an OMERO server. OMERO.matlab comes with a few conveniences for
 making this work.
 
-If you run ``client = loadOmero();`` (i.e. loadOmero with an output argument),
+If you run ``client = loadOmero();`` (i.e. loadOmero without an input argument),
```

Comment:
I see why the formulation was confusing and the change was made but note the original description still applies. Passing an output argument is still mandatory for the connection to be made or said otherwise `loadOmero()` will *not* create a connection - see
https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroM/src/loadOmero.m#L100